src/pages/tokenDrop/Payment.tsx

```diff
@@ -42,12 +44,23 @@ const Payment: FunctionComponent = () => {
   const dispatch = useDispatch();
 
   const bundlePrice = useGetMursPrice();
```

Comment:
We should remove references to "murs" in case we re-use this code for other artists.